src/components/shared/Button.js

```diff
@@ -36,11 +49,45 @@ const CustomButton = styled(GrommetButton)`
     `}
 `
 
-export const Button = ({ ...props }) => {
+export const Button = ({
+  large = false,
+  link = false,
+  responsive = false,
+  uppercase = false,
+  width,
+  ...props
+}) => {
+  const { viewport } = useResponsive()
   return (
-    <Box width="max-content">
-      {/* eslint-disable-next-line react/jsx-props-no-spreading */}
-      <CustomButton {...props} />
+    <Box
+      responsive={responsive}
+      width={
+        width || (responsive && viewport === 'small' ? '100%' : 'max-content')
+      }
+      viewport={viewport}
+    >
+      {link ? (
+        <GrommetButton
+          link={link}
```

Comment:
should this be href?

Reply:
This `link` prop is used for a button that is rendered as an underlined plain text with a font size of 14px, thank you! 

e.g. ) Line 34 of [SearchCardHeaderCTA](https://github.com/AlexsLemonade/refinebio-web/blob/bb5c9f5bccd20cbc0a2de9e46481a2b1f300ea37/src/components/SearchCard/SearchCardHeaderCTA.js) (previously DatasetCTA)
<img width="235" alt="image" src="https://user-images.githubusercontent.com/31800566/217141166-7d71dce5-b586-437b-9c82-1a44d978065f.png">